src/index.html

```diff
@@ -22,6 +22,391 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="Moyo"
```

Comment:
For logos, use the format: "[Organization name] logo." 

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/04636f4d-463c-4586-89d7-bd768d1aa8f7)


```suggestion
            alt="Moyo logo"
```